lab-exercise-1/NapassawanFeature.java

```diff
@@ -0,0 +1,5 @@
+public class NapassawanFeature {
+    public void feature() {
+        // this is a new feature
```

Comment:
This content is not well-written. please improve it.

Reply:
I think This content is not well-written. please improve it.

Reply:
Go go improve

Reply:
it not complete the feature, can you add the feature in your code.

Reply:
good